models/employee.py

```diff
@@ -78,6 +80,25 @@ def update_from_dict(self, d):
         self.department = d.get('department')
         self.meta_department = get_meta_department(self.department)
 
+    def get_gravatar(self):
+        """Creates gravatar URL from email address."""
+        email = '{user}@{domain}'.format(user=self.username, domain=config.DOMAIN)
```

Comment:
A quick code search shows that the rest of the code seems to use `employee.user.email()` ([eg](https://github.com/Yelp/love/blob/master/logic/love.py#L68)), so I think I'll actually do that instead.